src/components/ElementSearchDialog/element-search-dialog.tsx

```diff
@@ -54,8 +53,8 @@ const ElementSearchDialog = (props: ElementSearchDialogProps) => {
 
     const displayedValue = useMemo(() => {
         return searchTermDisabled || searchTermDisableReason
-            ? searchTermDisableReason
-            : searchTerm;
+            ? searchTermDisableReason ?? 'search disabled'
```

Comment:
manage translation for this default value

Reply:
done